core/common/src/ContextRealityModel.ts

```diff
@@ -15,6 +15,7 @@ import { FeatureAppearance, FeatureAppearanceProps } from "./FeatureSymbology";
  * @alpha
  */
 export interface OrbitGtBlobProps {
+  rdsUrl: string;
```

Comment:
You have added a new property, which will not be present in older data; therefore it needs to be marked optional (`rdsUrl?: string`).

Reply:
Sure, thanks, I'll include that.

Reply:
I've made rdsUrl an option rdsUrl?